rasa_nlu/data_router.py

```diff
@@ -35,6 +54,15 @@ def __init__(self, config, component_builder):
         self.emulator = self.__create_emulator()
         self.component_builder = component_builder if component_builder else ComponentBuilder(use_cache=True)
         self.model_store = self.__create_model_store()
+        self.pool = ProcessPool(config['max_training_processes'] or 1)
```

Comment:
Right that makes sense the question was more about this though:
There is default value for the configuration variable in the config.py, meaning it will **always** be set to that value if the user doesn't explicitly sets a different value. Therefore, the above default value is only used if the user explicitly sets `"max_training_processes": 0`. Which we can check for but if we do handle invalid input there it should be handling any user set value `< 0` instead of only `== 0`.

Gist, I don't see an added value in putting another default value here. Does that make sense?